ads/google/doubleclick.js

```diff
@@ -79,6 +79,14 @@ function doubleClickWithGpt(global, data, gladeExperiment) {
     parseInt(data.overrideHeight || data.height, 10),
   ]];
 
+  // Center the ad in the container.
+  const container = global.document.querySelector('#c');
```

Comment:
We already make this assumption for every GPT Light request which is running at 90% (see a few lines up from changes there).
I think this is a fairly safe assumption if people aren't explicitly removing this from their remote.html files. Perhaps the AMP folks would be better at saying what the policy here should be.
If we decide to consider this unsafe, we should change this for both GPT and GPT Light.